packages/design-mui/src/components/MuiFilter/MuiFilter.tsx

```diff
@@ -33,7 +34,55 @@ interface PopoverProps extends MuiFilterRowProps {
   clearFilter: () => void
 }
 
+interface Option {
+  operator: string
+  text: string
+}
+
+interface OperatorOption {
+  [key: string]: Option[]
```

Comment:
this can be merged into 1 type with `interface Option` if `Option` is not used anywhere else, also I think this can be more strictly typed: `key` here can only be `string` or `date` right? we can enforce that and expand it as we add more filter types

Reply:
Good call on making it more strict. Kept `Option` separate to be the return type of `getOperator`